### ISSUE TITLE
1.19.14 Ocarina of Time Master Quest - Purple loading screen, black screen in game

### ISSUE THREAD

**Issue by cchulo** [libretro/dolphin#247]
Latest dolphin core. I figured out the solution to this bug: set XFBToTextureEnable = false in the game's ini file.

However this only works once.

To reproduce this error, set  D43E01.ini in User/GameSettings folder to:

``` ini
[Video_Hacks]
XFBToTextureEnable = False
ImmediateXFBenable = False
```

Start retroarch, with the dolphin core boot up OoT MQ. You will see that the purple screen is fixed, and in game the pause menu does not present a black screen.

If you close the content, and DO NOT close Retroarch, but once again boot up OoT MQ, you will see the purple loading screen is back as well as the black pause menu. The only way to get it working again is to completely close Retroarch and reopen it.

These are just a cosmetic bug though, nothing game breaking. It's perfectly playable, but these artifacts, especially the black pause menu, are a bit distracting.

**Comment by bslenul** [comment]
Yeah there's a super annoying bug where closing content keeps the core alive in the background (invisible to the user), which can create all sorts of issues like the one you described, it can also be options not applying properly, aspect ratio being wrong on the next boot, etc. :/

**Comment by cchulo** [comment]
> Yeah there's a super annoying bug where closing content keeps the core alive in the background (invisible to the user), which can create all sorts of issues like the one you described, it can also be options not applying properly, aspect ratio being wrong on the next boot, etc. :/

is this dolphin core related or retroarch itself you think?

**Comment by bslenul** [comment]
I'd say a core issue, Flycast had a similar issue months ago but it was fixed and now it unloads properly on close content so it's probably fixable for Dolphin as well. Now we need someone interested and motivated enough to take a look at the issue... 😅

**Comment by cchulo** [comment]
I can take a crack at it, might take me a while though, never contributed before
